src/main/java/org/broadinstitute/hellbender/tools/exome/CreatePanelOfNormals.java

```diff
@@ -39,13 +40,38 @@
  * </p>
  *
  * @author Valentin Ruano-Rubio &lt;valentin@broadinstitute.org&gt;
+ *
+ * <h3>Examples</h3>
+ * <p>
+ *     The command encompasses empirically determined parameters for TCGA project data.
```

Comment:
Our imperative is to nudge users to determine the parameters that work best for their own data and not for them to blindly think what we give are best practices (which they are not).

The most informative thing we can give our users is the representative technology that the parameters were determined to work on, e.g. data circa 2012 or TCGA-type. I'm informed that most of the CNV test data is TCGA.

I opt to keep these statements. If you feel strongly otherwise @LeeTL1220, please take this up with @samuelklee and also inform me what data type was used to determine the parameters.